<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPSPointsWizardDialog</class>
 <widget class="QDialog" name="GPSPointsWizardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import datas...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_fileShow">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Name match option</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_importByName">
        <property name="text">
         <string>Import by image name</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Import by sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>571</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Separator</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,1,3">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox_tab">
        <property name="text">
         <string>tab</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_semicolon">
        <property name="text">
         <string>semicolon</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_comma">
        <property name="text">
         <string> comma</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_space">
        <property name="text">
         <string>space</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_other">
        <property name="text">
         <string>orthers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_other">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Row from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_rowFrom">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_multiAsSingle">
       <property name="text">
        <string>Treat consecutive delimiters as one</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>198</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>checkBox_tab</tabstop>
  <tabstop>checkBox_semicolon</tabstop>
  <tabstop>checkBox_comma</tabstop>
  <tabstop>checkBox_space</tabstop>
  <tabstop>checkBox_other</tabstop>
  <tabstop>lineEdit_other</tabstop>
  <tabstop>spinBox_rowFrom</tabstop>
  <tabstop>checkBox_multiAsSingle</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_tab</sender>
   <signal>clicked()</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>checkEnablePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_semicolon</sender>
   <signal>clicked()</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>checkEnablePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_comma</sender>
   <signal>clicked()</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>checkEnablePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_space</sender>
   <signal>clicked()</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>checkEnablePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_other</sender>
   <signal>clicked()</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>checkEnablePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>validImport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_rowFrom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>checkEnablePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_multiAsSingle</sender>
   <signal>stateChanged(int)</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>checkEnablePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_other</sender>
   <signal>editingFinished()</signal>
   <receiver>GPSPointsWizardDialog</receiver>
   <slot>checkEnablePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>preview()</slot>
  <slot>checkEnablePreview()</slot>
  <slot>validImport()</slot>
  <slot>onSelectCoordinate()</slot>
 </slots>
</ui>
